frontend/packages/console-shared/src/components/dashboard/activity-card/activity-card.scss

```diff
@@ -26,6 +26,13 @@
     var(--pf-v6-c-card--child--PaddingBlockEnd) var(--pf-v6-c-card--child--PaddingInlineStart);
 }
 
+.co-activity-card__recent-footer {
+  border-top: var(--pf-t--global--border--width--divider--default) solid
+    var(--pf-t--global--border--color--default);
+  padding: 1.5rem var(--pf-v6-c-card--child--PaddingInlineEnd)
+    var(--pf-v6-c-card--child--PaddingBlockEnd) 0;
+}
+
```

Comment:
use `    <Divider />` instead of CSS https://www.patternfly.org/components/card/#card-with-dividers